src/main/resources/lib/hudson/test/js/failureSummary.js

```diff
@@ -18,3 +18,24 @@ function hideFailureSummary(id) {
     document.getElementById(id + "-showlink").style.display = "";
     document.getElementById(id + "-hidelink").style.display = "none";
 }
+
+
+document.addEventListener('DOMContentLoaded', (event) => {
+
+    const testShowlinks = document.querySelectorAll("a[id*=-showlink]");
+    testShowlinks.forEach((element) => {
+        element.onclick = (_) => {
+            const testId = element.id.replace('-showlink', '');
+            showFailureSummary(testId, document.URL + "/summary");
```

Comment:
`document.URL` is wrong, as it refers to that page we're in. Request has to go to `JENKINS_URL/job/fs/5/testReport/<package_name>/<test_name>/summary` and not just `JENKINS_URL/job/fs/5/testReport/summary`
Currently this returns 404:
![image](https://user-images.githubusercontent.com/91559310/198654379-834829d9-c8b5-49a4-8971-8bc17cc7ebce.png)

Reply:
Ok, can you please complete the **Reproduction steps** section of the [Jira ticket (JENKINS-69658)](https://issues.jenkins.io/browse/JENKINS-69658) description ?

Reply:
ping @yaroslavafenkin

Reply:
Perfect! :)